src/content/en/2020/fonts.md

```diff
@@ -276,34 +317,50 @@ Lots more information about color fonts, including examples, are available at [c
 
 ## Variable fonts
 
+{{ figure_markup(
+  caption="Usage of variable fonts on mobile.",
+  content="11.00%",
+  classes="big-number",
+  sheets_gid="91894176",
+  sql_file="04_14a.variable_font_comparison_fcp.sql"
+) }}
+
+Variable fonts are certainly one of the biggest stories this year. They're seen in 10.54% of desktop pages, and 11.00% of mobile. That's up from an average of 1.8% last year, a huge growth factor. It is not hard to see why their popularity is increasing – they offer more design flexibility, and also potentially smaller binary font sizes, especially if multiple styles of the same font are used on the same page.
+
+{# That's huge usage! Any ideas what's driven it? It would appear no variable font has any where near that usage so how come total usage is so high? Has any particular service or platform turned this on by default? #}
```

Comment:
@raphlinus / @jpamental thoughts?

Reply:
The usage is largely driven by Google Fonts: they've started serving variable versions of a number of fonts where appropriate, without the web authors every knowing or having to do anything! (Which is a fantastic proof of the viability of the technology in my opinion)

Any time a site has enough different weights in use, GF will serve the variable version instead of static ones. They started doing that with Oswald in 2019, and it's grown to about 8-10 others now as well.

Reply:
Nice! Heard you talk in Paris at DotCSS (I think!) on the experiments on Oswald but wasn't aware they'd moved beyond this.

I think this is worth noting in the chapter. WDYT?

Reply:
I think that would be totally worth it. Maybe @davelab6 could jog my memory about which fonts they're serving now? (I can't remember where that conversation happened... too many message platforms!)